cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/BuildCommand.java

```diff
@@ -155,6 +155,10 @@ public BuildCommand() {
     @CommandLine.Option(names = "--cloud", description = "Enable cloud artifact generation")
     private String cloud;
 
+@CommandLine.Option(names = "--showAllWarnings", description = "Show all the warnings including the " +
```

Comment:
Apply for other applicable areas as well.

```suggestion
    @CommandLine.Option(names = "--showAllWarnings", description = "Show all the warnings including the " +
```